adapters/handlers/rest/configure_api.go

```diff
@@ -426,8 +424,15 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 	return appState
 }
 
-func parseNode2Port(appState *state.State) (m map[string]int, err error) {
-	m = make(map[string]int, len(appState.ServerConfig.Config.Raft.Join))
+// discoverNodes2Port it will collect nodes info from configrued RAFT_JOIN, RAFT_PORT
```

Comment:
typo(non-blocking): `s/configrued/configured` also the spacing of the comment seems a bit off below :)